torch/_dynamo/eval_frame.py

```diff
@@ -97,16 +97,16 @@ class Unset(Enum):
 
 unset = Unset.token
 
-
 def _maybe_set_eval_frame(callback: DynamoCallback):
     # A wrapper on set_eval_frame that is guarded by a Justknob.
     # Users can disable torchDynamo by setting the JK to False.
     from torch._C._dynamo.eval_frame import set_eval_frame
 
     if not justknobs_check("pytorch/compiler:enable_compiler_set_eval_frame"):
-        log.warning(
-            "Dynamo disabled by Justknob: enable_compiler_set_eval_frame, skipping set_eval_frame"
-        )
+            disable_warn = True
```

Comment:
dead code

Reply:
err, why does mypy/ruff not complain this is useless

Reply:
Derp fixed.